src/screens/OperationDetails/Footer.js

```diff
@@ -34,7 +46,7 @@ function Footer({ url, urlWhatIsThis }: Props) {
           event="OperationDetailViewInExplorer"
           type="primary"
           title={<Trans i18nKey="operationDetails.viewInExplorer" />}
-          onPress={() => Linking.openURL(url)}
+          onPress={viewInExplorer}
```

Comment:
why track in the onPress callback when you already track this with the button click :thinking: 
I think we can move this to not do the event is already present